index.html

```diff
@@ -184,16 +184,16 @@
     <h1 id = "title">Microverse Survey Form</h1>
```

Comment:
kindly use a different folder for the Css code

Reply:
- [ ] Kindly always include an explanation, or a "why", for each requested change, this will make your comments more useful and clear
- [ ] Also as I can see, this specific inline comment does not touch on any CSS file. Kindly be specific in the requested changes as right now another student would not know which file this is that you want them to create.